src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write code here
+function getNumberFromSalary(item) {
+  const result = +[...item.dataset.salary].map((char) => {
+    if (isNaN(char)) {
+      return;
+    }
+
+    return char;
+  }).join('');
+
+  return result;
+}
+
+function getEmployees(arr) {
+  const result = [];
+
+  arr.forEach((el) => {
+    result.push({
+      name: el.innerText,
+      position: el.dataset.position,
+      salary: el.dataset.salary,
+      age: el.dataset.age,
+    });
+  });
+
+  return result;
```

Comment:
use Array.map() instead forEach

Reply:
i don`t want mutate array, 1 variable = 1 data.

Reply:
why forEach is bad in this task?

Reply:
For the fact that you actually operate in memory with 2 arrays, one is iterated over and you push values into the other at each iteration of the cycle. The map method is not a mutating method, refresh your knowledge with this [link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map)

Please correct the code according to the recommendations
Also if you have any questions or if you didn't understand the mentor's comment, please ask them in fe_chat and tag mentors.